README.md

```diff
@@ -79,29 +65,38 @@ Click the **Deploy Azure** button to  begin.
+<a target="_blank" href="https://portal.azure.com/">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
 1. Select the Microsoft Azure account you want to access.
 1. Enter your Password and click **Sign in.**
-1. On the Customized Template page create a new Resource group by selecting the **Create New** icon and entering a name for the group. The Resource Group should be empty when you access the page.
-    * You can also select a pre-defined Resource group by selecting the **Use Existing** icon and clicking on one of the groups from the dropdown menu.
-4. Select a location from the dropdown menu.
-5. Enter a Username for the **domainAdminUsername**. This is a new account.
-6. Enter a password for the **domainAdminPassword**. This is a new password.
-7. Enter a **domainName** and ensure it finishes in **.com**.
-8. Enter your **AzureAdminUsername**. This must be the same account you logged into from step 1.
-9. Enter your **AzureAdminPassword**. This must be the same password you used to log in from step 2.
-10. Enter the CAS license registration code for the **registrationCode**.
-11. Use the default addresses that are pre-entered for the **adminVMBlobSource** and **_artifactsLocation**. 
-12. Read the Terms and Conditions and once you are satisified with the information you have entered click the **I Agree** icon.
-13. Click **Purchase** to begin deployment.
+1. Launch an Azure Cloud Shell Powershell instance. See [Overiew of Azure Cloud Shell](https://docs.microsoft.com/en-ca/azure/cloud-shell/overview?view=azurermps-5.0.0) for details on how to set this up. The rest of the setup steps will be run in the Azure Cloud Shell. It is recomended to maximize the Cloud Shell window so prompts can be properly seen.
+1. Change to your Cloud Drive directory. 
+    ```
+    cd $HOME\CloudDrive\
+    ```
+1. Download the CAM Deployment script.
+    ```
+    Invoke-WebRequest -URI https://raw.githubusercontent.com/teradici/deploy/bd/Deploy-CAM.ps1 -OutFile .\Deploy-CAM.ps1
+    ```
+1. Run the CAM Deployment script.
+    ```
+    .\Deploy-CAM.ps1
+    ```
+1. You may now be prompted to select a subscription to use to deploy CAM in. Choose the number accordingly.
+1. Exisiting Resource Groups will be listed if there are Resource Groups available for CAM to be deployed into. Either select an existing Resource Group by selecting the number next to desired Resource Group or create a new Resource Group by providing a name for the new Resource Group.
+1. Available deployment locations will be listed. Select a location from one of the listed options by typing in the *Location* field for the desired location.
+1. Enter the desired credentials for the **Administrative Domain User**. This is a new account that will be created.
+1. Enter a **Domain Name** and ensure it finishes in *.com*.
```

Comment:
or .something - How are we tying this into the parameters list, above? Is this the text from the admin guide? (I was always uncomfortable with just saying .com but I can't recall if that's in the admin guide or not right now...)

Reply:
That is currently what is in the admin guide. Is it not the case that it _has_ to be .com?

Reply:
This is pretty much what the original comment in the README said. I'll try to remove the ambiguity

Reply:
I'm probably mixing my memories of the different versions of this text - There's some verbiage in Deploy-CAM.ps1 which might help (or need fixing) as well.